webapp/packages/plugin-data-viewer/src/TableViewer/TableFooter/TableFooterMenu/TableFooterMenuItem.tsx

```diff
@@ -70,6 +70,7 @@ export const TableFooterMenuItem = observer(function TableFooterMenuItem({
         as="button"
         {...props}
         disabled={menuItem.isDisabled}
+        title={translate(menuItem.toolTip || '')}
```

Comment:
strange, if we translate an empty string, we get `title=''` property on an item, is it expected?